doc/manual/noteworthy-differences.rst

```diff
@@ -6,6 +6,39 @@
 Noteworthy Differences from other Languages
 *******************************************
 
+Noteworthy and unusual Julia features
+---------------------------------
+
+* Julia allows identifiers with complex Unicode, for example you could represent the value of a chi-squared test with ``χ²``,
+  similarly ``χ⁽³⁾`` is used in electromagnetism for the third-order nonlinear susceptibility, another example is ``∇²``
+  which is the name of the Laplacian operator. All of these Unicode symbols are valid Julia identifiers.
+
+  It can also be useful for caching powers of ``x`` to ensure that intermediate products are reused optimally in time-critical code. 
```

Comment:
I get why this example was put here, but this is a microoptimization that is going to freak people out if its the first thing they see in a section of the manual. I'm not sure an example is needed.